src/core-api/file/file.ts

```diff
@@ -1,4 +1,4 @@
-import * as FormData from 'form-data'
+import FormData from 'form-data';
```

Comment:
Have you checked that this works?

Reply:
* as did not work actually. compilation error

Reply:
That's weird. This is even tested code. But OK for now